source/extensions/filters/common/rbac/engine_impl.cc

```diff
@@ -9,8 +9,8 @@ namespace Common {
 namespace RBAC {
 
 namespace {
-const Envoy::Http::HeaderMapImpl empty_header = Envoy::Http::HeaderMapImpl();
-const envoy::api::v2::core::Metadata empty_metadata = envoy::api::v2::core::Metadata();
+const Envoy::Http::HeaderMapImpl empty_header;
```

Comment:
I suppose these should actually be constructed-on-first-use.

Reply:
yeah I have a change for it, up to you to change this here or I will open another PR